core/src/main/scala/better/files/WriterOutputStream.scala

```diff
@@ -41,7 +41,7 @@ class WriterOutputStream(writer: Writer, decoder: CharsetDecoder, bufferSize: In
       flushImmediately = flushImmediately
     )
 
-  override def write(b: Array[Byte], off: Int, len: Int) = {
+  override def write(b: Array[Byte] | Null, off: Int, len: Int) = {
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.